tests/std/tests/P0768R1_spaceship_operator/test.cpp

```diff
@@ -187,7 +179,101 @@ void test_algorithm() {
     assert((test_algorithm2<Ty1, Ty2>()));
     assert((test_algorithm2<Ty2, Ty1>()));
 }
-#endif // defined(__cpp_impl_three_way_comparison) && defined(__cpp_lib_concepts)
+#endif // __cpp_lib_concepts
+
+// Guard against regression of GH-1050: 0 <=> partial_ordering::unordered returns invalid value
+static_assert(!(std::partial_ordering::less == 0));
+static_assert(std::partial_ordering::less != 0);
+static_assert(std::partial_ordering::less < 0);
+static_assert(!(std::partial_ordering::less > 0));
+static_assert(std::partial_ordering::less <= 0);
+static_assert(!(std::partial_ordering::less >= 0));

```

Comment:
We should also have tests for literal `0` on the left-hand side, given that this selects different overloads and doesn't go through centralized logic. Currently, only `0 <=> MEOW` is tested.

Reply:
I think I'd prefer to make the symmetry manifest by having the `(_Literal_zero, meow)` overloads delegate to the `(meow, _Literal_zero)` overloads.

Reply:
> > Currently, only 0 <=> MEOW is tested.
>
> I think I'd prefer to make the symmetry manifest by having the `(_Literal_zero, meow)` overloads delegate to the `(meow, _Literal_zero)` overloads.

That's great, Casey, but it doesn't address the actual comment. Go ahead and be explicit about delegation of the `_Literal_zero` overloads as you like, and then add `meow <=> 0` tests.

Reply:
> That's great, Casey, but it doesn't address the actual comment. Go ahead and be explicit about delegation of the `_Literal_zero` overloads as you like, and then add `meow <=> 0` tests.

That does seem to be what the comment is asking for, but given that there are already both `meow <=> 0` and `0 <=> meow` tests, I think my first interpretation - that the comment was asking for e.g. `0 < meow` tests - was the correct one.

Reply:
Yes, I was asking for `0 < MEOW` (and `0 > MEOW` etc.) tests.

Reply:
I notice this conversation isn't resolved: would you still like `0 < MEOW` tests despite that those overloads now directly delegate to the `MEOW > 0` overloads? I'm not concerned, but it's five minutes work so I don't mind, either.

Reply:
Yes, I would. There's a nonzero chance we're going to say "hey, there are extra function calls in debug mode" and modify this code later.